views/components/ConfigurationsList/index.jsx

```diff
@@ -0,0 +1,851 @@
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          credentials: 'same-origin',
+          body: JSON.stringify({
+            disable: configID,
+          }),
+        })
+        .then(() => {
+          return
+        })
+    }
+  }
+  const handleCrumbs = () => {
+    setState((prevState) => {
+      return {
+        ...prevState,
+        snackTime: false,
+        uploadSnack: false,
+      }
+    })
+  }
+  const removeConfig = (configs, index, configID, ownsConfig) => {
+    updateConfigurations(configID, ownsConfig)
+    setState((prevState) => {
+      return {
+        ...prevState,
+        configurations: update(configs, {
+          $splice: [[index, 1]],
+        }),
+        snackTime: true,
+      }
+    })
+    if (index == state.preferences['config']) {
+      updateUserPreferences(0, 'index')
+    }
+  }
+  const openSearchUsers = (index, configID, shared, owner) => {
+    setState((prevState) => {
+      return {
+        ...prevState,
+        searchUsers: true,
+        selectedConfig: {
+          _id: configID,
+          index: index,
+        },
+        shared: shared.map((friend) => ({
+          label: friend,
+          value: friend,
+        })),
+        configOwner: owner,
+      }
+    })
+  }
+  const closeSearchUsers = () => {
+    setState((prevState) => {
+      return {
+        ...prevState,
+        searchUsers: false,
+        selectedConfig: {
+          _id: '',
+          index: -1,
+        },
+        shared: [],
+        configOwner: '',
+      }
+    })
+  }
+  const copyConfig = (config) => {
+    let newConfig = {}
+    newConfig['owner'] = user.uid
+    newConfig['readers'] = [user.uid]
+    newConfig['created'] = new Date().toUTCString()
+    newConfig['type'] = config['type']
+    newConfig['name'] = config['name']
+    newConfig['config'] = config['config']
+
+    return window
+      .fetch(`${basePath}/api/v1/users/${user.uid}/configs`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'same-origin',
+        body: JSON.stringify({
+          add: newConfig,
+        }),
+      })
+      .then((response) => {
+        if (response.status == 201) {
+          fetchConfigurations(user.uid)
+        }
+      })
+  }
+
+  const generateCards = (configs, preference) => {
+    let cards = []
+    if (configs && configs.length > 0) {
+      for (let item in configs) {
+        let ownsConfig = user.uid === configs[item]['owner'] ? true : false
+        let showTime =
+          'modified' in configs[item]
+            ? configs[item]['modified']
+            : configs[item]['created']
+        let localTime = moment.utc(showTime).local().format()
+        let updated = moment(localTime).calendar()
+        cards.push(
+          <Card
+            style={{
+              margin: '3px',
+            }}
+          >
```

Comment:
Ideally we should use css for this but we can at least 1-line this.

```suggestion
          <Card style={{ margin: '3px' }}>
```

Reply:
799d5fa